ludwig/modules/optimization_modules.py

```diff
@@ -13,50 +13,57 @@
 # limitations under the License.
 # ==============================================================================
 from dataclasses import asdict
-from typing import Optional
+from typing import Dict, Optional, Tuple, Type, TYPE_CHECKING
 
 import torch
 
-from ludwig.distributed.base import DistributedStrategy
-from ludwig.schema.optimizers import BaseOptimizerConfig, GradientClippingConfig, optimizer_registry
 from ludwig.utils.misc_utils import get_from_registry
 from ludwig.utils.torch_utils import LudwigModule
 
+if TYPE_CHECKING:
+    from ludwig.schema.optimizers import BaseOptimizerConfig, GradientClippingConfig
+
+
+def create_clipper(gradient_clipping_config: Optional["GradientClippingConfig"]):
+    from ludwig.schema.optimizers import GradientClippingConfig
```

Comment:
Do we need to import again if it's already imported under TYPE_CHECKING?

Reply:
`TYPE_CHECKING` doesn't import it at all, it's only used for static analysis of the type hints.

Reply:
Ah sorry, my bad